src/base.js

```diff
@@ -13,25 +13,55 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+import os from 'os';
 import { Command, flags as flagParser } from '@oclif/command';
+import { getConfig } from './utils/config';
+import print from './utils/print';
+
+export const defaultConfigFolder = '.lisk';
 
 export default class BaseCommand extends Command {
 	async init() {
 		const { flags } = this.parse(this.constructor);
 		this.flags = flags;
-	}
 
-	async catch(error) {
-		this.log({ error });
+		process.env.XDG_CONFIG_HOME =
+			process.env.LISK_COMMANDER_CONFIG_DIR ||
+			`${os.homedir()}/${defaultConfigFolder}`;
+		this.userConfig = getConfig(process.env.XDG_CONFIG_HOME);
 	}
 
 	async finally(error) {
 		if (error) {
-			this.error(error);
+			this.error(error.message ? error.message : error);
 		}
 	}
+
+	print(result, readAgain = false) {
+		if (readAgain) {
+			this.userConfig = getConfig(process.env.XDG_CONFIG_HOME);
+		}
+		print({
+			json: this.userConfig.json,
+			pretty: this.userConfig.pretty,
+			...this.flags,
+		}).call(this, result);
+	}
 }
 
+const jsonDescription =
+	'Prints output in JSON format. You can change the default behaviour in your config.json file.';
+
+const prettyDescription =
+	'Prints JSON in pretty format rather than condensed. Has no effect if the output is set to table. You can change the default behaviour in your config.json file.';
+
 BaseCommand.flags = {
-	json: flagParser.boolean(),
+	json: flagParser.boolean({
+		char: 'j',
+		description: jsonDescription,
+		allowNo: true,
+	}),
+	pretty: flagParser.boolean({
+		description: prettyDescription,
```

Comment:
Is there a reason not to add the `allowNo` feature here?

Reply:
We can, it was just not there, so I didn't put it for now, but i will add them =)